articles/policies/global-limit.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Global Rate Limit Policy For Authentication API
+description: This page details Authentication API Rate Limit Policy for free subscribers
+toc: true
+topics:
+    - auth0-policies
+    - rate-limits
+    - testing
+contentType:
+  - reference
+useCase:
+  - support
+---
+# Authentication API: Global Rate Limit Policy
```

Comment:
I can't find the document right now but I seem to remember reading (and talking to others) about how our SLA covers the Authentication API only.

So we may want to be careful about calling these sets of endpoints the Authentication API as that might cause confusion if we use a different definition here than elsewhere.

Reply:
How would you call it?

Reply:
Firstly, I'm not 100% sure having "Authentication API" in there is a problem, just raising it here for discussion.

What if the short paragraph below was called "Free Tenants Global Rate Limit Policy" and at the same level as https://auth0.com/docs/policies/rate-limits#endpoints-with-rate-limits?

Then this could just be the "Free Tenants Global Rate Limit Policy" details page.